src/cli/keyboard-distance.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Can be used to calculate the closest character from a list of possible one for a single character.
+ *
+ * Assumes that the keyboard that is used is a Qwerty keyboard.
+ *
+ * Used to guess the correct character for a cli option.
+ *
+ * @param {string} char - The character to find the closest match for.
+ * @param {string[]} possible - The list of possible characters.
+ *
+ * @returns {string} - The closest match.
+ */
+export default function keyboardDistance(char, possible) {
+    if (char.length > 1) {
+        throw new Error('First argument must be a single character.');
+    }
+
+    possible = possible || [];
+    const qwerty = [
+        ['1', '2', '3', '4', '5', '6', '7', '8', '9', '0'],
+        ['q', 'w', 'e', 'r', 't', 'y', 'u', 'i', 'o', 'p'],
+        ['a', 's', 'd', 'f', 'g', 'h', 'j', 'k', 'l'],
+        ['z', 'x', 'c', 'v', 'b', 'n', 'm']
+    ];
+
+    const getPosition = (c) => {
+        let val = {};
+        qwerty.forEach((row, rowIndex) => {
+            row.forEach((column, columnIndex) => {
+                if (c.toLowerCase() === column) {
+                    val = {
```

Comment:
`forEach` does not allow for early return here, will rewrite to a more performant implementation.